specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -51,6 +51,555 @@
         }
       }
     },
+    "AcsRouterJobCancelledEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.RouterJobCancelled event",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsRouterJobEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "note": {
+          "description": "Router Job Note",
+          "type": "string"
+        },
+        "dispositionCode": {
```

Comment:
Should this be an enum that uses x-ms-enum with "modelAsString"= true?
Example - https://github.com/Azure/azure-rest-api-specs/blob/1858dae34fbc8a93f390fa12f8d81231ab744ec0/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L1049-L1055

Reply:
Still missing the enum values.

Reply:
Actually, dispositionCode should be a string. Even though we have some defaults, the expectation is to have it set by customers. thats why you see i removed the enum attribute.